conf/messages.cy

```diff
@@ -233,16 +230,14 @@ help.webchat.nonaccessiblecontent.h2bullets=Non compliance with the accessibilit
 help.webchat.nonaccessiblecontent.li1=If you''re using a screen reader you may need to tab past the footer of the webpage to find the webchat. This fails WCAG section 1.3.2 Meaningful Sequence
 help.webchat.nonaccessiblecontent.li2=There are some error messages in this service that may not be descriptive enough for some users with visual impairments, although they are compatible with common assistive technology tools. This fails WCAG section 3.3.3 Error Suggestion
 help.webchat.nonaccessiblecontent.li3=To find the privacy and accessibility links, users should tab beyond the message input box. This fails WCAG section 2.4.3 Focus Order
-help.webchat.nonaccessiblecontent.li4=Some screen readers may fail to tell the user whether the print-end chat menu is open or not. This fails WCAG section 4.1.2 Name, Role, Value
-help.webchat.nonaccessiblecontent.li5=Jaws stops reading out the chat conversation when the user is in the message input box. Users should click ‘tab button’ and click up arrow
-help.webchat.nonaccessiblecontent.li6=Users of Jaws may find they receive no notification that the agent has sent a message. This fails WCAG section 4.1.2 Name, Role, Value
 
 help.webchat.howwetested.h2=How we tested this webchat
-help.webchat.howwetested.p1=The service was last tested on 18/02/2020 and was checked for compliance with the WCAG 2.1 AA standard.
-help.webchat.howwetested.p2=The service was built using parts that will be tested by the Digital Accessibility Centre. The full service was tested by HMRC and included a disabled user.
-help.webchat.howwetested.p3=This page was prepared on 18/02/2020.
+help.webchat.howwetested.p1=The service was last tested on 05/03/2020 and was checked for compliance with the WCAG 2.1 AA standard.
```

Comment:
Ensure this file lines up to the english message file, atm it does not

Reply:
Done